Kernel/WallGoMatrix.m

```diff
@@ -259,6 +398,26 @@ List of particle indices (obtainable via PrintFieldRepPositions[]), where each p
 ";
 
 
+ImportMatrixElements::usage = 
```

Comment:
This docstring is not used.
<img width="398" height="137" alt="Screenshot 2025-10-17 at 12 32 39" src="https://github.com/user-attachments/assets/7b7af434-58b6-408b-97fa-c0084a0af9f8" />
The reason is because a few lines below the docstring is overwritten with an empty string:
```
ImportMatrixElements::usage="";
```